src/Form.jsx

```diff
@@ -704,6 +705,37 @@ function Form({
                     isFilled={isFilled}
                 />
             ))}
+            {activeStep.buttons.map((field, i) => {
+                const hoverStyles = {};
+                const selectedStyles = {};
+                if (field.hover_border_color)
```

Comment:
move this logic into the ButtonElement component. Also if you're naming this ButtonElement, you should rename Text to TextElement

Reply:
Done.